### PR TITLE
Install thin-provisioning-tools on Debian

### PR DESCRIPTION
**Pull request by noelmcloughlin**

### CODE REVIEW

File: iscsi/osfamilymap.yaml [hunk 1]
```diff
@@ -42,6 +42,7 @@ Debian:
         - linux-image-generic
```
Comment (javierbertoli) [inline iscsi/osfamilymap.yaml]: `linux-image-generic` is a virtual package which does not exist in Debian anymore. Also, virtual packages support [be deprecated in SaltStack's next version](https://github.com/saltstack/salt/pull/47263). 

Would  you consider removing it from the list in this PR?

Comment (noelmcloughlin) [reply iscsi/osfamilymap.yaml]: Hi @javierbertoli thanks for reviewing - yes I have removed `linux-image-generic` from the list.

File: iscsi/osfamilymap.yaml [hunk 2]
```diff
@@ -39,9 +39,9 @@ Debian:
     pkgs:
       wanted:
         - linux-image-extra-virtual
```
Comment (javierbertoli) [inline iscsi/osfamilymap.yaml]: Huh, `linux-image-extra-virtual` isn't either a Debian package. Now that this one shows up, they are *Ubuntu*-specific. Mind you remove this one too, so we merge this PR?

Comment (noelmcloughlin) [reply iscsi/osfamilymap.yaml]: Thanks for clarifying that.  done!

**Comment by javierbertoli** [comment]
Merged. Thanks, @noelmcloughlin !

**Comment by noelmcloughlin** [comment]
I appreciate your review @javierbertoli  thx

**Pull request merged**